aries-core/src/js/components/core/Tiles/stories/tileHeader.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Tile, Tiles } from 'aries-core';
+import { storiesOf } from '@storybook/react';
+
+import {
+  Box,
+  Button,
+  Footer,
+  Grommet,
+  Header,
+  Heading,
+  Paragraph,
+} from 'grommet';
+import { More } from 'grommet-icons';
+import { hpeFont } from './prototype';
+
+export const TileHeader = () => (
+  <Grommet theme={hpeFont} full>
+    <Tiles margin="large">
+      <Tile height="medium" width="medium">
+        <Header pad="small">
+          <Heading level={4} size="xsmall">
+            Custom Header
+          </Heading>
+          <Button icon={<More color="control" />} onClick={() => {}} />
+        </Header>
+        <Box pad="small">
+          <Paragraph>
+            This custome Tile includes custom properties of: height, width &
+            Heading. This Tile is built with the Grommet components of Header &
+            Footer.
+          </Paragraph>
+        </Box>
+        <Box flex />
+        <Footer pad="small" background="light-4">
```

Comment:
Seems like by default, a Tile has rounded corners. With the Footer added, the rounding doesn't occur anymore on the bottom corners but it does still occur on the top corners. Do we want the rounding to be maintained by default here?

Reply:
This is just a story example, so we can go either way. 
Regarding:
>With the Footer added, the rounding doesn't occur anymore on the bottom corners
This statement is only true if you are using a background color for the footer.

Reply:
Gotcha, what if I want to have a background color but maintain the rounded bottom? Will that be supported

Reply:
of course, set `round` on Footer. https://v2.grommet.io/box#round

Reply:
I would think setting `round` on a Tile would set `overflow="hidden"`, so that any enclosed content wouldn't exceed the Tile.

Reply:
I looked at the designs, and it actually makes sense. Let me come up with a quick PR for Grommet Footer.

Reply:
@ericsoderberghp @taysea , something like this https://github.com/grommet/grommet/pull/3617/?

Reply:
I think it belongs on Tile, not on Footer.

Reply:
@ericsoderberghp, Not sure what you mean, Tile doesn't have a Footer component.
The thing we can do for this story example is to add `round` to Footer as suggested on top.

Reply:
I pushed an example of how this could be achieved via Footer on the Tile.